package.json

```diff
@@ -67,6 +68,7 @@
     "webpack-dev-server": "^3.8.0"
   },
   "dependencies": {
+    "@openmrs/esm-api": "^1.0.2",
```

Comment:
noticed for esm-navigation it's `^1.0.3`, does this mean these apps will be using different minor versions of esm-api?  should that be acceptable?

Reply:
I published 1.0.3 after I created this PR. I'm going to update this PR to 1.0.3, but here's an explanation of why it actually isn't necessary to do so:

The `dependencies` in the package.json are providing two things: (1) Typescript defns + autocomplete, (2) ability to package up a a frontend directory for the WAR file.

Regarding (1), 1.0.3 fixes a typescript problem that is not relevant for esm-home (since the build for esm-home is passing on 1.0.2).

Regarding (2), since 1.0.3 falls matches the semantic version range `^1.0.2`, I am pretty sure that we'll still only end up with one version of esm-api when `npm install`-ing both esm-home and esm-navigation. So packaging up a WAR file would still work out fine, I think.

All that said, I did upgrade to 1.0.3 just now. My point is just that keeping all the packages on exactly the same version of every package is not strictly necessary.